IPython/core/interactiveshell.py

```diff
@@ -193,8 +193,8 @@ def __repr__(self):
         name = self.__class__.__qualname__
         raw_cell = ((self.raw_cell[:50] + '..')
                     if len(self.raw_cell) > 50 else self.raw_cell)
-        return '<%s object at %x, raw_cell="%s" store_history=%s silent=%s shell_futures=%s result=%s>' %\
-               (name, id(self), raw_cell, store_history, silent, shell_futures, repr(self.result))
```

Comment:
This class doesn't have `self.result` - can you remove that one? Otherwise this looks good, thanks!

Reply:
Thanks, @takluyver! I did it in the second commit (sorry, I should have squashed them)

Reply:
Bah, sorry, my mistake. I thought you were adding it instead of taking it away. Must be tired!